sklearn/metrics/_pairwise_distances_reduction/_radius_neighborhood.pxd

```diff
@@ -0,0 +1,152 @@
+cimport numpy as cnp
```

Comment:
The pairwise part has been added by mistake I think.

Reply:
Ooo yeah not sure why rebasing got this. I'll just redo the changes from main.